src/Client.php

```diff
@@ -190,18 +196,18 @@ public function prepareOAuth()
                     $this->oauthConfig,
                     $this->getSignableParameters()
                 );
-                $requestHeaders = $this->getRequest()->getHeaders();
+                $requestHeaders   = $this->getRequest()->getHeaders();
                 $requestHeaders->addHeaders(['Authorization' => $oauthHeaderValue]);
                 break;
             case OAuth::REQUEST_SCHEME_POSTBODY:
-                if ($this->getRequestMethod() == HttpRequest::METHOD_GET) {
+                if ($this->getMethod() === HttpRequest::METHOD_GET) {
```

Comment:
is original `getRequestMethod()` cause error? If not, I prefer to keep original as is as unrelated to PR

Reply:
This is an invalid change. It _should_ be changed to:

```suggestion
                if ($this->oauthConfig->getRequestMethod() === HttpRequest::METHOD_GET) {
```

Because there is a magic `__call` in `Client` that proxies to `$this->oauthConfig`

Furthermore, there is no test coverage of these paths.

Reply:
Removing my approval here - nicely spotted @samsonasik